docs/operational_guide/repairs.md

```diff
@@ -0,0 +1,34 @@
+# Background Repairs (beta)
+
+## Overview
+
+An M3DB cluster can be configured to passively repair itself in the background. If the background repairs feature is enabled, then M3DB nodes will passively scan each other's metadata and when a mismatch is detected the nodes will perform a repair such that each node in the cluster eventually settles on a consistent view of the data.
+
+## Configuration
+
+The feature can be enabled by adding the following configuration to `m3dbnode.yml` under the `db` section:
+
+```yaml
+db:
+	... (other configuration)
+	repair:
+		enabled: true
+```
+
+In addition, the following two optional fields can also be configured:
+
+```yaml
+db:
+	... (other configuration)
+	repair:
+		enabled: true
+		throttle: 10s
+		checkInterval: 10s
+```
+
+The `throttle` field controls how long the M3DB node will pause between repairing each shard/blockStart combination and the `checkInterval` field controls how often M3DB will run the scheduling/prioritization algorithm that determines which blocks to repair next. In most situations operators should omit these fields and rely on the default values.
+
+## Caveats and Limitations
```

Comment:
Is there a minimum number of nodes required in a cluster for this to work?

Reply:
No I think it will just be a no-op if there is only one node in the cluster